kem/hybrid/ckem.go

```diff
@@ -1,57 +1,72 @@
 package hybrid
 
-// TODO move over to crypto/ecdh once we can assume Go 1.20.
-
 import (
-	"crypto/elliptic"
+	"crypto/ecdh"
 	cryptoRand "crypto/rand"
-	"crypto/subtle"
-	"math/big"
 
 	"github.com/cloudflare/circl/kem"
 	"github.com/cloudflare/circl/xof"
 )
 
 type cPublicKey struct {
 	scheme *cScheme
-	x, y   *big.Int
+	key    *ecdh.PublicKey
 }
 type cPrivateKey struct {
 	scheme *cScheme
-	key    []byte
+	key    *ecdh.PrivateKey
 }
 type cScheme struct {
-	curve elliptic.Curve
-}
-
-var p256Kem = &cScheme{elliptic.P256()}
-
-func (sch *cScheme) scSize() int {
-	return (sch.curve.Params().N.BitLen() + 7) / 8
+	curve ecdh.Curve
 }
 
-func (sch *cScheme) ptSize() int {
-	return (sch.curve.Params().BitSize + 7) / 8
-}
+var p256Kem = &cScheme{ecdh.P256()}
```

Comment:
Let's make a `cScheme{}` struct to satisfy the `kem.Scheme` interface, instead of the pointer to a `cScheme`.

Thus, `cPrivateKey` and `cPublicKey` does not store a pointer to `cScheme`.